libs/components/src/table/table-data-source.ts

```diff
@@ -65,7 +83,7 @@ export class TableDataSource<T> extends DataSource<T> {
 
   private updateChangeSubscription() {
     const filteredData = combineLatest([this._data, this._filter]).pipe(
-      map(([data, filter]) => this.filterData(data, filter))
+      map(([data, filter]) => this.filterData(data))
```

Comment:
```suggestion
      map(([data]) => this.filterData(data))
```

We won't need filter here if we are removing it from the method's parameters.